README.md

```diff
@@ -1 +1,9 @@
-# DigitalBankingAbstracta
\ No newline at end of file
+# DigitalBankingAbstracta
+
+To run the tests:
```

Comment:
Assume the reader of this readme doesn't necessarily have anything in their environment and they're just cloning your project (this is tipically the scenario your readme users are going to be at). Therefore include:

- Prerequisites (they need at least node.js and docker)
- They won't have the dependencies needed, how do they install them? (basically running `npm install` on project's root)
- How do they host the digital bank application? (They have to clone it, then spin up the services with docker-compose, etc)

Also, try to use markdown syntax to make it easier to read , like `##` for headers/sections, `1. Element` for bullets, and "back quotes" for code formatting, for example